package.json

```diff
@@ -39,6 +39,7 @@
   },
   "scripts": {
     "test": "sk-tests",
+    "test-watch": "sk-tests-watch",
```

Comment:
Nit: are really only for automated npm stuff; the `sk-*` commands are public API from `skatejs-build`.